src/pybind/mgr/dashboard/osd_perf.html

```diff
@@ -121,26 +121,34 @@
 
 <section class="content">
 
-    <table>
-        <tr>
-            <td>
-                <h3>Writes</h3>
-                <table id="heatmap_write">
-                </table>
-            </td><td>&nbsp;</td><td>
-                <h3>Reads</h3>
-                <table id="heatmap_read">
-                </table>
-            </td>
-        </tr>
-    </table>
+    <div class="row">
+        
+        <div class="col-md-4">
+            <h3>Writes</h3>
+            <table id="heatmap_write">
+            </table>
+        </div>
+
+        <div class="col-md-4">
+            <h3>Reads</h3>
+            <table id="heatmap_read">
+            </table>
+        </div>
+            
+    </div>
+
+<hr>
+
```

Comment:
@tchaikov  I tried to remove the empty lines, but they are not showing up here. Ironically, they are removed in my local repo and the file is committed too. Can you please help me out in this?